### PR TITLE
Prevent accidential selection of line numbers in code view

### PR DESCRIPTION
**Pull request by silverwind**
Before:

nums-before

After:

nums-after

### CODE REVIEW

**Comment by appleboy** [comment]
@silverwind Please also update the `css` file.

**Comment by silverwind** [comment]
Done, but an unrelated change got built in the css.

**Comment by silverwind** [comment]
Might be better to untrack the css from git :)

**Comment by appleboy** [comment]
@silverwind I can take it but thanks for your update.

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

**Comment by silverwind** [comment]
@appleboy I'll look into overhauling the assets build.

**Comment by appleboy** [comment]
@silverwind move discussions to #1861

**Comment by lafriks** [comment]
LGTM if for now you could manualy add back removed css part so that PR code would be cleaner

**Comment by silverwind** [comment]
> LGTM if for now you could manualy add back removed css part so that PR code would be cleaner

Just land the two commits seperately.

**Comment by appleboy** [comment]
@silverkorn Please add back removed css part since you don't remove any source code from `less` file.

**Comment by silverwind** [comment]
Found the issue, my local branch was out of date and did not include https://github.com/go-gitea/gitea/pull/1288. After rebasing, `make assets` did the right thing.

File: public/css/index.css [hunk 1]
```diff
@@ -1405,6 +1405,10 @@ footer .ui.language .menu {
   color: #999;
   background: #f5f5f5;
   width: 1%;
+  -moz-user-select: none;
+  -ws-user-select: none;
```
Comment (geek1011) [inline public/css/index.css]: There's no such thing. It should be `-ms-user-select`, not `-ws-user-select`.

File: public/less/_repository.less [hunk 2]
```diff
@@ -275,6 +275,10 @@
 					color: #999;
 					background: #f5f5f5;
 					width: 1%;
+					-moz-user-select: none;
+					-ws-user-select: none;
```
Comment (pgaskin) [inline public/less/_repository.less]: There's no such thing. It should be `-ms-user-select`, not `-ws-user-select`.

File: public/css/index.css (hunk 1 above)
Comment (pgaskin) [inline public/css/index.css]: There's no such thing. It should be `-ms-user-select`, not `-ws-user-select`.

**Review by pgaskin — APPROVED** [review]
Looks good other than my comments.

**Comment by silverwind** [comment]
@geek1011 fixed that typo. I wish we would use Autoprefixer to prevent issues like that :)

**Comment by silverkorn** [comment]
@appleboy , Thanks for the typo in the username, you sent me a notice and now I know about this project! 😆 
Adding a star right now!

**Comment by appleboy** [comment]
@silverkorn  Sorry for tag wrong name. 👍 I hope I'm not bothering you.

**Pull request merged**